src/Stack/index.jsx

```diff
@@ -10,22 +10,30 @@ const DIRECTION_VARIANTS = [
 const Stack = forwardRef(({
   direction,
   gap,
+  reverse,
   children,
   className,
   ...rest
-}, ref) => (
-  <div
-    ref={ref}
-    className={classNames(
-      direction === 'horizontal' ? 'pgn__hstack' : 'pgn__vstack',
-      gap ? `pgn__stack-gap--${gap}` : '',
-      className,
-    )}
-    {...rest}
-  >
-    {children}
-  </div>
-));
+}, ref) => {
+  const childrenArray = React.Children.toArray(children);
+
+  if (reverse) {
```

Comment:
Since we already use flex boxes here, can we reverse order with them instead of js? I think it would be better performance-wise and overall more consistent with how we designed this component.

You just need to override `flex-direction` property, e.g.:
```SCSS
.pgn__vstack {
  flex: 1 1 auto;
  flex-direction: column;

  &.pgn__stack-reversed {
    flex-direction: column-reverse;
  }
}

.pgn__hstack {
  flex-direction: row;
  align-items: center;

  &.pgn__stack-reversed {
    flex-direction: row-reverse;
    justify-content: flex-end;
  }
}
```

and then add `pgn__stack-reversed` class to the className based on `reversed` prop (same way we do it with `gap`).